tests/components/directv/test_media_player.py

```diff
@@ -1,398 +0,0 @@
-"""The tests for the DirecTV Media player platform."""
```

Comment:
Why are we removing these tests?

Reply:
these tests relied a lot upon the way the previous incarnation of the integration was written and media player was excluded from code coverage up until I enabled in my initial update to this integration. The new module I wrote for this now has tests of it's own against its request and API functions. The media player is now simply calling or using data from the module rather than making decisions on it's own so these tests no longer make sense

Reply:
It still makes sense to text the state services of the media_player. The only thing that should need updating is the mocking of the library.

Reply:
I've added some tests back